templates/c_profile.html

```diff
@@ -5,103 +5,148 @@
+                              <label class="form-label">Job Ranking: </label>
+                              <input type="text" class="form-control" value="">
+                          </div>
+                          <div class="form-group">
+                              <label class="form-label">Ethnicity: </label>
+                              <input type="text" class="form-control" value="">
+                          </div>
+                      </div>
                     </div>
-        
-                    <div class="tab-pane fade" id="change-login-info">
-                      <div class="card-body pb-2">
-
+                  </div>
+                  <div id="tab2" class="tab-content">
+                    <div id="change-login-info">
+                      <div class="card-body">
                         <div class="form-group">
                             <label class="form-label">Username: </label>
                             <input type="text" class="form-control" value="">
                         </div>
-        
-                        <div class="form-group">
-                            <label class="form-label">Current password: </label>
-                            <input type="password" class="form-control">
-                        </div>
-            
                         <div class="form-group">
                             <label class="form-label">New password: </label>
                             <input type="password" class="form-control">
                         </div>
-            
                         <div class="form-group">
                             <label class="form-label">Repeat new password: </label>
                             <input type="password" class="form-control">
                         </div>
-        
+                      </div>
+                    </div>
                   </div>
                 </div>
-                </div>
-            <div class="text-right mt-3">
-                <button type="button" class="btn btn-primary">Save changes</button>&nbsp;
-                <button type="button" class="btn btn-default">Cancel</button>
+              </div>
+              <div class="text-right mt-9">           
+                <button type="submit" form="ChangeAccInfoForm">Save changes</button>&nbsp;
+                <button type="reset">Cancel</button>
+              </div>
             </div>
-    
+        </div>
       </div>
-    </div>
+    <script>
+      /*document.getElementById("change-account-info").tab("show")
+      $('.list-group-flush account-settings-links a[href="#change-login-info"]').tab("show")*/
+
+      // Scroll bar (need to be modified)
+      /*function scrollToTop() {
+        window.scrollTo(0,0)
+      }*/
+
+      // Default Tab
+      // Get the element with id="defaultpage" and click on it
+      window.addEventListener('load', function() {
+        document.getElementById('defaultpage').click();

```

Comment:
Easiest option to avoid scrolling page up manually every time you open it- remove this event listener